package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "kat-footer",
-  "version": "0.0.1",
+  "version": "2.0.3",
```

Comment:
You shouldn't need to update this manually. We should be able to do a tagged build which would update this